ClearKeep/Scenes/Call/CallView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CallView.swift
+//  ClearKeep
+//
+//  Created by đông on 05/04/2022.
+//
+
+import SwiftUI
+import Common
+
+struct CallView: View {
```

Comment:
Why need this view ?

Reply:
because PR 916 create after 912, this code is in 912, when 912 merged it will be fixed